patients/tide.go

```diff
@@ -25,7 +25,7 @@ type TideConfig struct {
 type TideFilters struct {
 	DropInTimeInTargetPercent *string `json:"dropInTimeInTargetPercent,omitempty"`
 	TimeCGMUsePercent         *string `json:"timeCGMUsePercent,omitempty"`
-	TimeInLowPercent          *string `json:"timeInLowPercent,omitempty"`
+	TimeInAnyLowPercent       *string `json:"timeInAnyLowPercent,omitempty"`
```

Comment:
This again could just be my ignorance, but should there be a `TimeInAnyHighPercent`?

Reply:
This is just the filters struct we send back via the API for the client to know the filters, its an early version of what it should be and just hardcodes in the values that were used for the categories.

We dont have a category using the `AnyHighPercent`, so it doesn't exist here for the API response.

A more flexible version already exists in the #117 PR.